src/app/services/artists.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from "@angular/core";
+import { HttpClient } from "@angular/common/http";
+import { Observable, forkJoin } from "rxjs";
+import { environment } from "src/environments/environment";
+import { Artists } from "../models/artists";
+import { Artist } from "../models/artist";
+import { mergeAll, toArray } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: "root"
+})
+export class ArtistsService {
+  constructor(private http: HttpClient) {}
+
+  getInitialArtists(): Observable<Artists> {
+    //Dummy query
+    return this.http.get<Artists>(`${environment.apiUri}/search/artist?q=deer`);
+  }
+
+getTopArtists(): Observable<Artist[]> {
+    let artists$: Observable<Artist>[] = [];
+
+    for (let i = 1; i < 22; i++) {
+      if (i === 12) {
```

Comment:
What is this check for?

Reply:
Fixed it now, but I originally put it in because it never returned an image, which we later learned was because of CORS rate limiting